Editor/AGS.Editor/Panes/ScriptEditor.cs

```diff
@@ -56,6 +56,19 @@ public ScriptEditor(Script scriptToEdit, AGSEditor agsEditor, Action<Script> sho
             this.Resize += new EventHandler(ScriptEditor_Resize);
         }
 
+        public string GetScriptTabName()
+        {
+            if (_roomNumber > 0)
+            {
+                UnloadedRoom room = Factory.AGSEditor.CurrentGame.FindRoomByID(_roomNumber);
+                if(room != null && room.Number > 0 && !string.IsNullOrEmpty(room.Description))
+                {
+                    return Script.FileName + ": " + room.Description + (IsModified ? " *" : "");

```

Comment:
I think it's better to have `*` mark right after filename, because when description is too long, the trailing part is hidden, and `*` will not be visible.

E.g. the tab name may look like: "room3.asc: Bla bla bla ..."

Reply:
fixed it in this push https://github.com/adventuregamestudio/ags/compare/49ca91d1d367bb35d6b0a7db9feff1b948423ebe..56ad1d6b2d3e0e5edc52c4d4f94f36c03e441544